compiler/rustc_trait_selection/src/traits/error_reporting/suggestions.rs

```diff
@@ -873,6 +879,57 @@ impl<'a, 'tcx> InferCtxtExt<'tcx> for InferCtxt<'a, 'tcx> {
         }
     }
 
+    fn suggest_remove_await(
+        &self,
+        obligation: &PredicateObligation<'tcx>,
+        err: &mut DiagnosticBuilder<'_>,
+    ) {
+        let span = obligation.cause.span;
+
+        if let ObligationCauseCode::AwaitableExpr(hir_id) = obligation.cause.code {
+            err.span_suggestion_verbose(
+                span,
+                "do not `.await` the expression",
```

Comment:
This sounds a tiny bit blunt. What do you think of this, which to me sounds a bit more like other rustc suggestions?

```suggestion
                "remove the `.await`",
```